docs/rich_presence/Using_with_the_Game_SDK.mdx

```diff
@@ -21,11 +21,12 @@ Before we dig in, it's helpful to understand what Rich Presence data you can set
 | location                               | field name     | notes                                                                       |
 |----------------------------------------|----------------|-----------------------------------------------------------------------------|
 | First row below title                  | details        |                                                                             |
-| Second row below title                 | state          |                                                                             |
-| Second row below title                 | partySize      | In parenthesis next to the `state`, first number in the format `(1 of 10)`  |
-| Second row below title                 | partyMax       | In parenthesis next to the `state`, second number in the format `(1 of 10)` |
-| Third row below title                  | startTimestamp | Converted to a format such as `01:33 elapsed`                               |
-| First button at the bottom             | joinSecret     | Button has the text "Ask to join"                                           |
+| Second row below title                 | state          | If provided with no `partySize` or `partyMax`                               |
```

Comment:
Isn't this row a duplicate of the "second badge in last row below title" one? Going by the image you uploaded ![](https://github.com/user-attachments/assets/1100b718-9a55-47b7-bd48-77116b112b4a) the state is the second badge in the last row regardless of being in a party or not.

Reply:
Yeah the second image is actually incorrect -- since the "bad example" has no party field, "In Lobby" should render in a line of text below the details string.

Following up on fixing this.

Reply:
Updated this to use a consistent `state (party)` string in both examples!